lib/media_types/scheme.rb

```diff
@@ -17,10 +19,50 @@
 require 'media_types/scheme/output_type_guard'
 require 'media_types/scheme/rules_exhausted_guard'
 
-require 'json'
-
 module MediaTypes
   class AssertionError < StandardError
+    def initialize(errors)
+      @fixture_errors = errors
+    end
+
+    def message
+      @fixture_errors.map { |error|
+        error.message
+      }.to_s
+    end
```

Comment:
```suggestion
      fixture_errors.map(&:message).join(', ')
    end
```

I think this is what ya meant.